web/packages/core/texts/es-ES/messages.ftl

```diff
@@ -14,6 +14,11 @@ enable-hardware-acceleration = Al parecer, la aceleración de hardware no esta h
 view-error-details = Ver los detalles del error
 open-in-new-tab = Abrir en una pestaña nueva
 click-to-unmute = Haz clic para dejar de silenciar
+clipboard-message-title = Para copiar y pegar en Ruffle
+clipboard-message-description = {{$variant}} Pero siempre se puede usar estos  atajos:
```

Comment:
Um, this is missing the values for the variant...

Reply:
I see. I'm really behind with my Discord reading nowadays...

The approach that worked for me is to copy the source string, and then only touch the parts that looked like they will actually end up appearing on the interface for the user. In this case, I indicated the place of those (partial) sentences with `...`:

```
{ $variant ->
   *[unsupported] ...
    [access-denied] ...
} ...
```

Reply:
I have submitted (and approved) a Spanish translation based partly on Google Translate, and partly on your submission, just so you have a working starting point. Feel free to fix it up!

Reply:
You can use the preview feature, and for `variant` under it, enter either `unsupported` or `access-denied` to see how the final composed string will look like for users.